src/Plugin/KeyProvider/ConfigKeyProvider.php

```diff
@@ -38,8 +38,8 @@ public function defaultConfiguration() {
   public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
     $form['key_value'] = array(
       '#type' => 'textarea',
-      '#title' => $this->t('Key value'),
-      '#description' => $this->t("Enter the key to save in Drupal's configuration system."),
+      '#title' => t('Key value'),
```

Comment:
so this should still be $this->t(). Extending \Drupal\Core\Plugin\PluginBase, instead of Component, will provide the t() method.